infra/testApi.mjs

```diff
@@ -97,6 +97,18 @@ const localError = await callHerbie("/api/localerror", {
 assertIdAndPath(localError)
 assert.equal(localError.tree['avg-error'] > 0, true)
 
+const json1 = `
+{"formula":"${FPCoreFormula}","sample":[[[2.852044568544089e-150],1e+308]],"seed":5}`
```

Comment:
Why format it this way? Kinda bizarre...

Reply:
I will fix and format like the other request. I copied it from Firefox.